integration/runtime_handler_test.go

```diff
@@ -38,15 +44,13 @@ func TestRuntimeHandler(t *testing.T) {
 		runtimeService.RemovePodSandbox(sb)
 	}()
 
-	t.Logf("Verify runtimeService.PodSandboxStatus sets RuntimeHandler")
+	t.Logf("Verify runtimeService.PodSandboxStatus sets runtimeHandler")
```

Comment:
`runtimeService.PodSandboxStatus() returns ...`

Reply:
If I understand correctly the resulting `sbStatus.RuntimeHandler` should be logged?

Would this be confusing for the empty `--runtime-handler` flag case as the output would then be `runtimeService.PodSandboxStatus returns `. This was why I initially removed the logging on lines 44 and 50 in favor of only logging what is happening at a high level but I can revert that change if desired.

Thanks for the feedback!

Reply:
yes just saying we are making sure the function returns the value that was originally set to runtimeHandler.. thus returns runtimeHandler.. if you will

Reply:
the "sets" action occurred earlier on